### ISSUE TITLE
error: Unsupported UNWIND_INFO version

### ISSUE THREAD

**Issue by williballenthin** [williballenthin/lancelot#190]
https://www.virustotal.com/gui/file/0000b60cadd40a47ba881513fe733371105112628e9436f60787c30beaef4021

```
2023-06-06 13:40:31 [ERROR]  Unsupported UNWIND_INFO version

Stack backtrace:
   0: anyhow::error::<impl core::convert::From<E> for anyhow::Error>::from
             at /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/anyhow-1.0.71/src/error.rs:547:25
   1: <T as core::convert::Into<U>>::into
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/core/src/convert/mod.rs:717:9
   2: lancelot::analysis::pe::runtime_functions::read_unwind_info
             at ./core/src/analysis/pe/runtime_functions.rs:95:20
   3: lancelot::analysis::pe::runtime_functions::find_pe_runtime_functions
             at ./core/src/analysis/pe/runtime_functions.rs:168:39
   4: lancelot::workspace::PEWorkspace::from_pe
             at ./core/src/workspace/mod.rs:116:32
   5: lancelot::workspace::workspace_from_bytes
             at ./core/src/workspace/mod.rs:395:25
   6: smoke::_main
             at ./bin/src/bin/smoke.rs:76:14
   7: smoke::main
             at ./bin/src/bin/smoke.rs:93:21
   8: core::ops::function::FnOnce::call_once
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/core/src/ops/function.rs:250:5
   9: std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/sys_common/backtrace.rs:134:18
  10: std::rt::lang_start::{{closure}}
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/rt.rs:166:18
  11: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/core/src/ops/function.rs:284:13
  12: std::panicking::try::do_call
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panicking.rs:485:40
  13: std::panicking::try
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panicking.rs:449:19
  14: std::panic::catch_unwind
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panic.rs:142:14
  15: std::rt::lang_start_internal::{{closure}}
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/rt.rs:148:48
  16: std::panicking::try::do_call
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panicking.rs:485:40
  17: std::panicking::try
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panicking.rs:449:19
  18: std::panic::catch_unwind
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/panic.rs:142:14
  19: std::rt::lang_start_internal
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/rt.rs:148:20
  20: std::rt::lang_start
             at /rustc/ce5919fcef67103098219e1868f741e56fc90963/library/std/src/rt.rs:165:17
  21: main
  22: __libc_start_call_main
  23: __libc_start_main@@GLIBC_2.34
  24: _start
```

**Comment by williballenthin** [comment]
the version is indicated as "2" which is not described here: https://learn.microsoft.com/en-us/cpp/build/exception-handling-x64?view=msvc-170

there's some public discussion here:
  - https://mingw-w64-public.narkive.com/VXMzxcux/unwind-info-version-2-any-documentation
  - https://sourceware.org/legacy-ml/gdb-patches/2013-12/msg00097.html

so i think we can add support for version 2, since we don't use the field that was redefined?

**Comment by williballenthin** [comment]
closed in bd87709
